impl/src/main/java/com/okta/sdk/impl/resource/DefaultFileResource.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2021-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.impl.resource;
+
+import com.okta.sdk.impl.ds.InternalDataStore;
+import com.okta.sdk.resource.FileResource;
+
+import java.io.File;
+import java.util.Collections;
+import java.util.Map;
+
+public class DefaultFileResource extends AbstractResource implements FileResource {
+
+    private final File file;
+    private final String formDataName;
+
+    public DefaultFileResource(InternalDataStore dataStore, File file, String formDataName) {
```

Comment:
Requiring the use of a "File" limits the use cases where this can be used.  A stream should be used.

For example, an application could be loading the image from a database, or another remote location, and would not have a "File" directly. 

Supporting files directly is still a good idea too, but a `Path` should be used instead of a `File`.

Reply:
Updated. now `java.nio.file.Path` used instead of `java.io.File`